COLLABORATOR_GUIDE.md

```diff
@@ -167,32 +213,37 @@ export default MyComponent;
 - Each Props type should be prefixed by the name of the Component
 - Components should always be the `default` export of a React Component file
-- Avoid using DOM/Web APIs/`document`/`window` API access within a React Component. Use utilities or Hooks when you need a Reactive state
+- Avoid using DOM/Web APIs/`document`/`window` API access within a React Component.
+  Use utilities or Hooks when you need a Reactive state
 - Avoid making your Component too big. Deconstruct it into smaller Components/Hooks whenever possible
 
 ## Unit Tests and Storybooks
 
-Each new feature or bug fix should be accompanied by a unit test (when deemed valuable). We use [Jest][] as our test runner and [React Testing Library][] for our React unit tests.
+Each new feature or bug fix should accompany a unit test (when deemed valuable).
+We use [Jest][] as our test runner and [React Testing Library][] for our React unit tests.
+
+We also use [Storybook][] to document our components.
+Each component should have a storybook story that documents the component's usage.
 
-We also use [Storybook][] to document our components. Each component should have a storybook story that documents the component's usage. Snapshot testing of our components is directly done by taking snapshot of all Storybook stories, using [Storybook Test Runner][] and [Playwright][].
+Visual Regression Testing is automatically done via [Chromatic](https://www.chromatic.com/) to ensure that Components are rendered correctly.
 
 ### General Guidelines for Unit Tests
 
 Unit Tests are fundamental to ensure that code changes do not disrupt the functionalities of the Node.js Website:
 
 - Unit tests should be written as `.mjs` files.
-- We recommend that unit tests are added for content covering `util`, `scripts`, `hooks` and `components` whenever possible.
-- Unit Tests should cover that the functionality of a given change is working as expected.
+- We recommend adding unit tests for content covering `util`, `scripts`, `hooks`, and `components` whenever possible.
+- Unit Tests should ensure that a given change's functionality is working as expected.
 - When creating unit tests for React components, we recommend that the tests cover all the possible states of the component.
-- We also recommend mocking external dependencies, if unsure about how to mock a certain dependency, raise the question on your Pull Request.
-  - We recommend using [Jest's Mock Functions](https://jestjs.io/docs/en/mock-functions) for mocking dependencies.
-  - We recommend using [Jest's Mock Modules](https://jestjs.io/docs/en/manual-mocks) for mocking dependencies that are not available on the Node.js runtime.
+- We also recommend mocking external dependencies, if unsure about how to mock a particular dependency, raise the question on your Pull Request.
+  - We recommend mocking dependencies by mocking [Jest's Mock Functions](https://jestjs.io/docs/en/mock-functions).
```

Comment:
this sentence read better before

Reply:
`We recommend mocking dependencies by mocking Jest's Mock Functions` uses the word mock three times. unresolved this to consider reverting the phrasing